decidim-core/app/views/decidim/devise/passwords/new.html.erb

```diff
@@ -1,31 +1,27 @@
 <% add_decidim_page_title(t("devise.passwords.new.forgot_your_password")) %>
 
-<div class="wrapper">
-<div class="row collapse">
-  <div class="row collapse">
-    <div class="columns large-8 large-centered text-center page-title">
-      <h1><%= t("devise.passwords.new.forgot_your_password") %></h1>
-    </div>
+<%= render layout: "layouts/decidim/shared/layout_center" do %>
```

Comment:
I'm totally unable to reach that view out. Initially, I styled `users/passwords/new`, but this login change I suppose is in `users/password/edit`, but I'm being always redirected to somewhere else (both logged in and out, no matter in local/staging/old_design), so I just edited it fully blindfolded

Reply:
It looks now fine (although with the same problems as the registration view but you already commented those):
![Password change view](https://user-images.githubusercontent.com/864340/183370025-46fd5419-6cc3-4edb-9c79-4c44bc1fca24.png)

You get to this view by
1. Filling in an email at `/users/password/new`
2. Going to `/letter_opener`
3. Clicking the "Change my password link" -> this will get you to `/users/password/edit` with a valid token

You cannot enter the password change view unless the URL has a valid password change token which is generated for the user when you initiate the password change.

Reply:
Ok, thanks, it looks like we've something wrong configurated in our both servers, due to `/letter_opener` does not display any updates :disappointed:

Reply:
Usually production/staging servers are configured with an actual SMTP server to send out the emails when you'd also have to use an actual email to receive those messages. Don't know how your staging is configured.

The `/letter_opener` only works on development by default:
https://github.com/PopulateTools/decidim-populate-redesign/blob/c68f5f196ed1ef613e62a6800700849a894eb51f/config/routes.rb#L2-L4

https://github.com/PopulateTools/decidim-populate-redesign/blob/c68f5f196ed1ef613e62a6800700849a894eb51f/config/environments/development.rb#L36